controller/AuthController.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+
+from flask import request, make_response, jsonify
+from controller import app
+from dao import UserRepo
+from dao.models import User
+from flask_jwt_extended import create_access_token
+
+"""
+API call for user signup
+"""
+
+
+@app.route('/signup', methods=['POST'])
+def signup():
+    body = request.get_json()
+    user = User(**body)
+    if len(body.get("password")) > 8:
+        user.hash_password()
+    else:
+        return make_response(jsonify(response='Password must contain at least 8 characters'), 400)
+    UserRepo.create_user(user)
+    return make_response(jsonify(response='User added successfully'), 201)
```

Comment:
The logic on line 22 continues only after the if, so the code can be simplified a little:
```suggestion
    if len(body.get("password")) < 8:
        return make_response(jsonify(response='Password must contain at least 8 characters'), 400)
    user.hash_password()
    UserRepo.create_user(user)
    return make_response(jsonify(response='User added successfully'), 201)
```